pages/multiple-choice/index.njk

```diff
@@ -10,7 +10,7 @@ eleventyExcludeFromCollections: true
 {% extends 'layout.njk' %}
 
 {% set seoTitle %}Question {{ questionGroup.pageNumber }} of {{ questionGroup.questionTotal }} | Multiple Choice - {{ questionGroup.category }} | Trivia11y{% endset %}
-{% set seoDescription %}Answer question {{ questionGroup.pageNumber }} of {{ questionGroup.questionTotal }} about {{ questionGroup.category }} (Multiple Choice).{% endset %}
+{% set seoDescription %}Answer question {{ questionGroup.pageNumber }} of {{ questionGroup.questionTotal }}: {{ questionGroup.category }} (Multiple Choice).{% endset %}
```

Comment:
I just wanted something other than "Answer question 14 of 40 about All Questions (Multiple Choice)". The colon felt like a good compromise that worked equally well for "All Questions" as well as something like "Forms".

Thinking about the context, though, the most likely case for someone reading the description is if the link is shared on social media. I wonder if what we have makes sense for that, or maybe it would be better to just have a generic description about Trivia11y as a whole 🤔 

I'm happy to call that out of scope for this, though!